downloader/src/main/kotlin/vcs/Cvs.kt

```diff
@@ -113,28 +113,41 @@ class Cvs : VersionControlSystem(), CommandLineTool {
                 return rootDir ?: workingDir
             }
 
-            private fun listSymbolicNames(): Map<String, String> {
-                val cvsLog = run(workingDir, "log", "-h")
-                var tagsSectionStarted = false
-
-                return cvsLog.stdout.lines().mapNotNull { line ->
-                    if (tagsSectionStarted) {
-                        if (line.startsWith('\t')) {
-                            line.split(':', limit = 2).let {
-                                Pair(it.first().trim(), it.last().trim())
+            private fun listSymbolicNames(): Map<String, String> =
```

Comment:
Please check the commit message, I don't understand what you mean by "surround".

Reply:
That refers to the `run(workingDir, "update", "-d")` and `finally` code which "surrounds" / wraps the `run(workingDir, "log", "-h")` call. Previously, that wrapping happened on a higher level in the caller of `listSymbolicNames()`, but it actually makes sense to move that "surrounding" / wrapping code closer to the code that needs to be wrapped.